dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/parser/TimePlaceholderUtils.java

```diff
@@ -363,6 +375,92 @@ private static String calculateTime(String expression, Date date) {
         return value;
     }
 
+    /**
+     * get week of year
+     * @param expression expression
+     * @param date       date
+     * @return week of year
+     */
+    public static String calcYearWeek(String expression, Date date) {
```

Comment:
```suggestion
    public static String calculateYearWeek(String expression, Date date) {
```

`calc` is a wired shorthand. I prefer to use `calculate` here.